CodeGeneration/Sources/SyntaxSupport/DeclNodes.swift

```diff
@@ -906,6 +906,14 @@ public let DECL_NODES: [Node] = [
       "WithTrailingComma"
     ],
     children: [
+      /// Indicates whether the 'without' operator was applied to the type to
+      /// indicate the suppression of implicit conformance to this type.
+      /// This child stores the token representing the 'without' operator.
+      Child(
+        name: "HasWithout",
```

Comment:
I would prefer this to be named `WithoutTilde`. This actually contains the tilde itself, so it’s not a boolean variable, which the `Has` implies.